docs/zh/start/rpc/dubbo_json_rpc.md

```diff
@@ -15,21 +15,16 @@ go build -o layotto cmd/layotto/main.go
 
 这里使用了`dubbo-go-samples`提供的示例服务
 ```sh
-git clone git@github.com:apache/dubbo-go-samples.git
+git clone git@github.com:apache/dubbo-go-samples.git (f0d1e1076397a4736de080ffb16cd0963c8c2f9d since Sun Mar 20 22:01:12)
 cd dubbo-go-samples
 
 # start zookeeper
-cd attachment/go-server
-make -f ../../build/Makefile docker-up 
-cd -
+cd rpc/jsonrpc/go-server
+docker-compose -f docker/docker-compose.yml up -d
 
-# build dubbo server
-cd general/jsonrpc/go-server
-sh assembly/mac/dev.sh
-
-# start dubbo server
-cd target/darwin/{generate_folder}/
-sh ./bin/load.sh start
+# build && start dubbo server
+cd cmd
+go run .
```

Comment:
I found this demo needs go v1.17. I ran into a problem when running this line with go v1.14:

```
go: downloading github.com/uber/jaeger-lib v2.4.1+incompatible
# golang.org/x/net/http2
/Users/qunli/go/pkg/mod/golang.org/x/net@v0.0.0-20211105192438-b53810dc28af/http2/transport.go:417:45: undefined: os.ErrDeadlineExceeded
note: module requires Go 1.17
```

So, could you please add a comment above this command (or at the beginning of this document) stating that this demo needs go v1.17 ?

Reply:
Well, I added Note at the beginning of this document.